client/js/lounge.js

```diff
@@ -147,6 +151,8 @@ $(function() {
 	});
 
 	socket.on("init", function(data) {
+		$("#loading-page-message").text("Rendering…");
```

Comment:
On slow device it might take a while to render all the channels and messages at once.